src/registrar/templates/admin/change_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends "admin/change_form.html" %}
+
+{% block object-tools %}
+{% if change and not is_popup %}
```

Comment:
Is this grabbed directly from the template? If so, did you see any other condition here, for instance: if change and is_popup?